lib/utils.js

```diff
@@ -32,6 +34,10 @@ function sort(arr) {
 function isEqual(a, b) {
   var i;
 
+  if (_.isNaN(a) && _.isNaN(b)) {
```

Comment:
could you add this to `isApproxEqual` as well, and add the note to the doc above?